src/modules/monitor.f90

```diff
@@ -75,7 +75,16 @@ module monitor
 
        real(kind=cgnsRealType), dimension(:,:,:), allocatable :: &
                                                                convArray
-!
+
+       ! iterArray(0:Itermax, sps, 4): stores the cumulative number of iterations, 
+       !                       CFL, step size, and lin res at each major iter.
+       !                       It is used post processing convergence histories
+       real(kind=realType), dimension(:,:,:), allocatable:: solverDataArray
```

Comment:
Right now the position of the variables in the convArray is sized to match the number of monitor variables and  is consistent with the order of monNames. 
I thought it would be cleaner to maintain this connection by separating the data.

Reply:
Right, that's what I had in mind, extend the monitor variables, but I guess that could be done in another PR.

Reply:
Is this the only comment left to address? Will appending to `convArray` mean that these other variables will also be available in the CGNS output (thanks to #189)? If so, I would be in favor of combining the arrays.

Reply:
We can also just modify how the data is written to the CGNS output too.
I'm still in favor of keeping them separate, since I think this will be easier to implement and more explicit.

There is no real performance benefit that I see to combining the arrays. 
I'd rather no add an nmon+4 to the loops over convArray, or remember/document whether the solver variables come at the beginning or end of the extended array.

Reply:
That's a fair point. I do think this is cleaner so I am fine this PR as is.